publisher-sdk/src/main/java/com/criteo/publisher/ConsumableBidLoader.java

```diff
@@ -21,15 +21,15 @@
 import com.criteo.publisher.model.CdbResponseSlot;
 import org.jetbrains.annotations.NotNull;
 
-public class InHouse {
+public class ConsumableBidLoader {
```

Comment:
I think it's worth documenting why this class has such a name (even if the class is internal). Today we interpret that `consuming` as: being able to use the `Bid` only once. But, if another person comes in this won't be entirely clear and she will think: what does an unconsumable bid mean?

Reply:
Done